FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -584,49 +584,77 @@ void test_coverage_vTaskSetTaskNumber_null_task_handle( void )
     /* Nothing will be changed. This test shows its result in the coverage report. */
 }
 
-/*
-The kernel will be configured as follows:
-    #define configNUMBER_OF_CORES                               (N > 1)
-    #define INCLUDE_uxTaskGetStackHighWaterMark             1
-
-Coverage for 
-    UBaseType_t uxTaskGetStackHighWaterMark( TaskHandle_t xTask )
-        By passing a valid created Task
-*/
-void test_task_get_stack_high_water_mark( void )
+/**
+ * @brief uxTaskGetStackHighWaterMark - return high water mark of a task.
+ *
+ * Verify the high water mark returned.
+ *
+ * <b>Coverage</b>
+ * @code{c}
+ * UBaseType_t uxTaskGetStackHighWaterMark( TaskHandle_t xTask )
+ * {
+ *     ...
+ * }
+ * @endcode
+ * xTask is not NULL.
+ */
+void test_coverage_uxTaskGetStackHighWaterMark( void )
 {
-    TaskHandle_t xTaskHandles[1] = { NULL };
+    TCB_t xTaskTCB = { NULL };
+    UBaseType_t uxReturn;
 
-    /* Create  tasks  */
-    xTaskCreate( vSmpTestTask, "SMP Task", configMINIMAL_STACK_SIZE, NULL, 2, &xTaskHandles[0] );
+    /* Setup the variables and structure. */
+    UnityMalloc_StartTest();
```

Comment:
you can add those (UnityMalloc_StartTest, UnityMalloc_EndTest) in the setup and teardown instead of repeating them here and there

Reply:
The following tasks won't pass the malloc count check. I am seeking Adam's help to fix them.
Can we merge this first and create another PR to move (UnityMalloc_StartTest, UnityMalloc_EndTest) in setup and teardown function?

- test_coverage_prvAddNewTaskToReadyList_create_two_tasks_with_the_first_suspended
- test_coverage_vTaskCoreAffinitySet_task_core_affinity_set_task_implied
- test_coverage_vTaskCoreAffinitySet_task_core_affinity_set_task_explicit
- test_coverage_vTaskCoreAffinitySet_task_core_affinity_change_while_running
- test_coverage_vTaskCoreAffinitySet_task_core_affinity_change_while_suspended
-